data/society/schools-libraries/index.html

```diff
@@ -42,20 +37,65 @@
 abstract="This data package contains a statewide schools dataset and a statewide libraries dataset." %}
 <div class="grid package">
   <div class="grid__col grid__col--12-of-12">
-    <h3 id="Schools">Schools</h3>
+    <h3 id="SchoolsPreKto12">Schools Pre K to 12</h3>
   </div>
   <div class="grid__col grid__col--12-of-12 package-content">
-    <img class="inline-text-left outline" src="{% link images/Schools.png %}" alt="Schools" />
-    <p>Society.Schools represents point locations of schools in the State of Utah. Schools of all levels from pre-kindgergarten to higher education are included. The sources for the list of schools in this dataset are the Utah State Office of Education and the Utah Education Network (UEN). AGRC used a combination of geocoding and manual placement (digitizing) to identify the geographic location of each school.</p>
-    <h5>Related Resources</h5>
+    <img class="inline-text-left outline" src="{% link images/Schools.png %}" alt="SchoolsPreKto12" />
+    <p>This dataset contains point locations for preschool and K-12 schools in the state of Utah - updated for the 2019-2020 school year. Higher education and adult vocational training are not included in the dataset.</p>
+    <p>The Schools Pre K to 12 dataset was built in collaboration with <a href="https://wfrc.org/">Wasatch Front Regional Council</a> (WFRC), the <a href="https://schools.utah.gov/">Utah State Board of Education</a> (USBE), and the <a href="https://gis.utah.gov/">Utah Automated Geographic Reference Center</a> (AGRC). The dataset includes public and private K-12 school locations with enrollment and demographics info by grade for public schools (charter and traditional) reported to the USBE. While some preschools are included, this is not a comprehensive statewide dataset for preschools.</p>
```

Comment:
**suggestion**: use real table name
**nitpick**: always use link tags for internal links. There should never be gis.utah.gov links in a page.

```suggestion
    <p>The <code>Schools_PreKto12</code> dataset was built in collaboration with <a href="https://wfrc.org/">Wasatch Front Regional Council</a> (WFRC), the <a href="https://schools.utah.gov/">Utah State Board of Education</a> (USBE), and the <a href="{% link index.html %}">Utah Automated Geographic Reference Center</a> (AGRC). The dataset includes public and private K-12 school locations with enrollment and demographics info by grade for public schools (charter and traditional) reported to the USBE. While some preschools are included, this is not a comprehensive statewide dataset for preschools.</p>
```